lib/inputs/distroless/static.ts

```diff
@@ -16,6 +16,12 @@ export function getAptFiles(extractedLayers: ExtractedLayers): string[] {
     if (!("dpkg" in extractedLayers[fileName])) {
       continue;
     }
+    // when the nodejs distroless image is build, the metadata is added to status.d
+    // this causes us to wronfgully identify nodejs as deb package
+    // https://github.com/GoogleContainerTools/distroless/blob/main/private/remote/node_archive.bzl#L29
+    if (fileName === "/var/lib/dpkg/status.d/nodejs") {
```

Comment:
Are you sure this is the correct approach? It works, but would this be a potential for false negatives in other images?

Reply:
Sorry, meant to close this PR - no, I don't think it's safe :) Was checking different approaches but everything I came across so far has the risk you mentioned, to replace a false positive with a false negative, which is worse. 
Closed it now